python/smqtk/algorithms/saliency/sbsm.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+from tqdm import tqdm
+import copy
+from PIL import Image
+import os
+from sklearn.metrics.pairwise import euclidean_distances
+import six
+from smqtk.algorithms import SmqtkAlgorithm
+from smqtk.algorithms.saliency import SaliencyBlackbox,ImageSaliencyAugmenter
+from smqtk.representation.data_element.memory_element import DataMemoryElement
+
+class SBSM_SaliencyBlackbox (SaliencyBlackbox):
+    
+    """
+    Blackbox function that produces some floating point scalar value for a
+    given descriptor element.
+    :param query_f: Feature of query image
+    :param type: smqtk.representation.DescriptorElement
+    :param base_descr: Base image descriptor
+    :param type: smqtk.representation.DescriptorElement
+        """
```

Comment:
This trailing ``"""`` seems to be indented an extra time over. Please just use a single indent here.

Reply:
There should also be a blank line between this and the ``__init__`` method.